daal4py/sklearn/_utils.py

```diff
@@ -362,7 +318,9 @@ def get_suggested_n_threads(n_cpus):
 
 
 def run_with_n_jobs(method):
-    """Decorator for running of methods containing oneDAL kernels with 'n_jobs'"""
+    """Decorator for running of methods containing oneDAL kernels with 'n_jobs'.
+    This decorator is not expected to be used manually.
```

Comment:
It might be better to write few words here about how it should be used then. If it shouldn't be used by user then it's probably worth to name it with leading underscore to highlight this fact.

Reply:
Description was extended